packages/moments/src/client/getMomentsDefaultClient.ts

```diff
@@ -0,0 +1,73 @@
+import { MomentsClient } from 'moments';
+import {
+  AuthenticationProvider,
+  AuthenticationProviderHttp,
+  PoapCompass,
+  PoapMomentsApi,
+} from '@poap-xyz/providers';
+
+interface GetMomentsDefaultClientOptions {
+  moments: {
+    momentsBaseUrl?: string;
+    /**
+     * The authentication provider used for authenticating requests to the moments API.
+     * If this is provided, oAuthAuthentication will be ignored.
+     */
+    authenticationProvider?: AuthenticationProvider;
+    /**
+     * The OAuth authentication configuration used for authenticating requests to the moments API.
+     * If this is provided, it will be used to create an `AuthenticationProviderOAuth` instance.
+     */
+    oAuthCredentials?: {
+      clientId: string;
+      clientSecret: string;
+      oAuthServerDomain?: string;
+    };
+  };
+  compass?: {
+    compassApiKey?: string;
```

Comment:
The API key in compass provider is required.

```suggestion
  compass: {
    compassApiKey: string;
```